zeromq/README.md

```diff
@@ -0,0 +1,60 @@
+# Overview
+
+This is zeromq 4.2.2 for rumprun. The test platform is running Ubuntu 16.04 and the compilation options include ```-std=c++17```.
+
+## Maintainer
+
+* Bela Berde, bela.berde@gmail.com
+* Github: @kvart2006
+
+## Instructions
+Use the included``Makefile`` by typing ```make```. Note that ```libtool``` is required to run ```autogen.sh```, and ```automake``` and ```autoconf``` for ```configure```.
```

Comment:
what is the expected outcome of the ``make``? For example, could you point out what are the executables or binaries out of the ``make`` command.